contracts/oracle/ChainlinkOracleWrapper.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "./IOracle.sol";
+import "../refs/CoreRef.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
+
+/// @title Chainlink oracle wrapper
+/// @author eswak
+/// @notice Reads a Chainlink oracle value & wrap it under the standard Fei oracle interface
+contract ChainlinkOracleWrapper is IOracle, CoreRef {
+    using Decimal for Decimal.D256;
+    using SafeMath for uint256;
+
+    /// @notice the referenced chainlink oracle
+    AggregatorV3Interface public chainlinkOracle;
+    uint256 public oracleDecimalsNormalizer;
```

Comment:
Lets add a comment saying this is assumed to be immutable with respect to Chainlink oracles per 4.12